### PR TITLE
Add .html to thredds urls

### PR DESCRIPTION
**Pull request by TAlonglong**
**Summary**:

**Related issue**: 

**Suggested reviewer(s)**:

**Reviewer checklist**:

- [x] The headers of all files contain a reference to the repository license (i.e., "License: This file is part of py-mmd-tools, licensed under the Apache License 2.0 (https://www.apache.org/licenses/LICENSE-2.0)")
- [x] 100% test coverage of new code - meaning:
    - [x] The overall test coverage increased or remained the same as before
    - [x] Every function is accompanied with a test suite
    - [x] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
    - [x] Functions with optional arguments have separate tests for all options
- [x] Examples are supported by doctests
- [x] All tests are passing
- [x] All names (e.g., files, classes, functions, variables) are explicit
- [x] Documentation (as docstrings) is complete and understandable

The checklist is based on the S-ENDA conventions and definition of done (see https://s-enda-documentation.readthedocs.io/en/latest/general_conventions.html). The above points are not necessarily relevant to all contributions. In that case, please add a short explanation to help the reviewer.

### CODE REVIEW

**Comment by TAlonglong** [comment]
Its still draft, but let me know of this is an acceptable approach, then I will update the test.

**Comment by TAlonglong** [comment]
I think I got it.

File: tests/test_check_mmd.py [hunk 1]
```diff
@@ -70,6 +72,17 @@ def test_all_urls_1(self):
         self.assertTrue(check_urls(['https://www.met.no']))
         self.assertFalse(check_urls(['http://met.not']))
 
+    # Check special urls
+    @patch('requests.get')
+    def test_special_urls_1(self, mock_get):
+        # Check urls with WMS
+        self.assertTrue(check_urls(['https://thredds.met.no/thredds/wms/remotesensingsatellite/polar-swath/2020/12/01/metopb-avhrr-20201201155244-20201201160030.nc?service=WMS&version=1.3.0&request=GetCapabilities']))
```
Comment (mortenwh) [inline tests/test_check_mmd.py:79]: problem med slike tester er at vi gjør oss avhengig av at serveren er oppe. Se på min test i tests_nc_to_mmd, linje 557 - test_file_on_thredds. Den passerer hvis serveren er nede...

File: tests/test_check_mmd.py [hunk 2]
```diff
@@ -70,6 +72,17 @@ def test_all_urls_1(self):
...
+
+    @patch('requests.head')
+    def test_special_urls_2(self, mock_head):
+        # Check urls with string dodsC
+        self.assertTrue(check_urls(['https://thredds.met.no/thredds/dodsC/remotesensingsatellite/polar-swath/2020/12/01/metopb-avhrr-20201201155244-20201201160030.nc']))
```
Comment (mortenwh) [inline tests/test_check_mmd.py:84]: samme her

**Review by mortenwh — CHANGES_REQUESTED** [review]

File: tests/test_check_mmd.py (hunk 1 above)
Comment (TAlonglong) [reply tests/test_check_mmd.py]: Nei, siden request.get er patchet kjøres ikke selve request kallet. Men jeg har endret testen til å bare ha de strengene vi skal teste for.

File: tests/test_check_mmd.py (hunk 2 above)
Comment (TAlonglong) [reply tests/test_check_mmd.py]: samme

File: tests/test_check_mmd.py (hunk 1 above)
Comment (mortenwh) [reply tests/test_check_mmd.py]: aha - la ikke merke til patchingen... sorry, det er bra!

**Review by mortenwh — APPROVED** [review]

**Pull request merged**